connector/dialect.tdd

```diff
@@ -420,6 +484,62 @@
       <argument type='real' />
       <argument type='int' />
     </function>
+    <function group='numeric' name='MOD' return-type='int'>
+      <formula>MOD(%1,%2)</formula>
+      <argument type='int' />
+      <argument type='int' />
+    </function>
+    <function group='numeric' name='MOD' return-type='real'>
+      <formula>MOD(%1,%2)</formula>
+      <argument type='real' />
+      <argument type='int' />
+    </function>
+    <function group='numeric' name='MOD' return-type='real'>
+      <formula>MOD(%1,%2)</formula>
+      <argument type='int' />
+      <argument type='real' />
+    </function>
+    <function group='numeric' name='MOD' return-type='real'>
+      <formula>MOD(%1,%2)</formula>
+      <argument type='real' />
+      <argument type='real' />
+    </function>
+    <function group='numeric' name='PI' return-type='real'>
+      <formula>3.141592653589793</formula>
+    </function>
+    <function group='numeric' name='POWER' return-type='int'>
+      <formula>POW(%1,%2)</formula>
+      <argument type='int' />
+      <argument type='int' />
+    </function>
+    <function group='numeric' name='POWER' return-type='real'>
```

Comment:
Similarly to my question for FLOOR, is there a specific reason to add this combination?